examples/c/sim_wt_model_nx6.c

```diff
@@ -32,6 +32,8 @@
 #include "acados/sim/sim_erk_integrator.h"
 #include "acados/sim/sim_irk_integrator.h"
 #include "acados/sim/sim_lifted_irk_integrator.h"
+#include <acados/sim/sim_gnsf.h>
```

Comment:
detail: use " instead of < for consistency with other headers

Reply:
(in principle, one should only need the header of the c interface of the integrators)

Reply:
> in principle, one should only need the header of the c interface of the integrators
It is needed to set the additional dimensions, and to use the `gnsf_import_matrices` and `gnsf_precompute`.
I tried to use the interface as far as possible, but these things cannot be easily made part of it..